<launch>
  <!-- Description -->
  <param name="robot_description" textfile="$(find str1ker)/description/robot.urdf" />

  <!-- Controller configuration -->
  <rosparam file="$(find str1ker_moveit_config)/config/ros_controllers.yaml" />

  <!-- ROS control hardware interface -->
  <node
    name="hardware"
    pkg="str1ker"
    type="hardware"
    output="screen"
  />

  <!-- Controller manager -->
  <node
    name="controller_spawner"
    pkg="controller_manager"
    type="spawner"
    respawn="false"
    output="screen"
    args="joint_state_controller arm_position_controller"
  />

  <!-- Robot state publisher -->
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="robot_state_publisher"
    respawn="false"
    output="screen"
  />
</launch>
